### PR TITLE
New package: libfixposix-0.4.3

### PR DESCRIPTION
**Pull request by TeddyDD**
Reqired for sequeler update

### CODE REVIEW

File: srcpkgs/libfixposix-devel [hunk 1]
```diff
@@ -0,0 +1 @@
+libfixposix/
```
Comment (maxice8) [inline srcpkgs/libfixposix-devel]: no leading slash

File: srcpkgs/libfixposix/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+homepage="https://github.com/sionescu/libfixposix"
+distfiles="https://github.com/sionescu/libfixposix/archive/v${version}.tar.gz"
+checksum=78fe8bcebf496520ac29b5b65049f5ec1977c6bd956640bdc6d1da6ea04d8504
+
+pre_configure() {
+	autoreconf -fi
+}
+
+pre_install() {
+	vlicense LICENCE
```
Comment (maxice8) [inline srcpkgs/libfixposix/template]: doesn't need vlicense

File: srcpkgs/libfixposix/template [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+libfixposix-devel_package() {
+	depends="${makedepends} ${sourcepkg}>=${version}_${revision}"
```
Comment (maxice8) [inline srcpkgs/libfixposix/template]: there are no makedepends and use `-` instead of `>=`

File: srcpkgs/libfixposix-devel (hunk 1 above)
Comment (TeddyDD) [reply srcpkgs/libfixposix-devel]: could you explain?

Comment (maxice8) [reply srcpkgs/libfixposix-devel]: no leading slash on the symlink.

```
cd srcpkgs
ln -s libfixposix libfixposix-devel
```

File: srcpkgs/libfixposix/template [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+license="BSL-1.0"
+homepage="https://github.com/sionescu/libfixposix"
+distfiles="https://github.com/sionescu/libfixposix/archive/v${version}.tar.gz"
+checksum=78fe8bcebf496520ac29b5b65049f5ec1977c6bd956640bdc6d1da6ea04d8504
+
+pre_configure() {
+	autoreconf -fi
+}
+
+pkg_install() {
```
Comment (maxice8) [inline srcpkgs/libfixposix/template]: pkg_install is only for subpackages. for the main package you're looking for do_install()

File: srcpkgs/libfixposix/template [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+	vcopy usr/lib/*.so usr/lib
+}
+
+libfixposix-devel_package() {
+	depends="${sourcepkg}-${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/lib/*.so
```
Comment (maxice8) [inline srcpkgs/libfixposix/template]: please quote this vmove statement

File: srcpkgs/libfixposix-devel (hunk 1 above)
Comment (TeddyDD) [reply srcpkgs/libfixposix-devel]: should be ok now

**Pull request closed without merging**